### ISSUE TITLE
active_record integration breaks without sinatra

### ISSUE THREAD

**Issue by hawknewton** [DataDog/dd-trace-rb#328]
Here's my config:
```ruby
Datadog.configure do |config|
  config.use :active_record, service_name: 'random-service'
  config.use :grape, service_name: 'random-service'
end

```
And here's what I get:

```
E, [2018-01-25T21:10:34.973666 #6271] ERROR -- ddtrace: [ddtrace] (/usr/local/var/rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/ddtrace-0.11.0/lib/ddtrace/contrib/active_record/patcher.rb:100:in `rescue in sql') 'sinatra' is not a valid integration.
```

I'm sure it has something to do with [this](https://github.com/DataDog/dd-trace-rb/blob/master/lib/ddtrace/contrib/active_record/patcher.rb#L61).  To be fair I haven't tried it with sinatra, but I'd wager it'll work.

**Comment by palazzem** [comment]
@hawknewton pretty sure it could be related. Thank you very much for reporting the issue, we're working on it immediately.

**Comment by palazzem** [comment]
@hawknewton PR has been submitted and reviewed! Once the CI is green, we'll merge the PR and it will be released with the next bugfix release (happening next Monday). Thanks for reporting this critical issue!

**Comment by palazzem** [comment]
@hawknewton merged! it will be part of the next 0.11.1 release! thanks again (also for the review)!
